test/Pattern/CallbackCacheTest.php

```diff
@@ -31,8 +32,9 @@ function bar(): string
 
 /**
  * @group      Laminas_Cache
+ * @property Cache\Pattern\CallbackCache $pattern
  */
-class CallbackCacheTestAbstract extends AbstractCommonPatternTest
+class CallbackCacheTest extends AbstractCommonPatternTestCase
```

Comment:
That's an OMG: no `test/Pattern/` test where ever executed because they had not the proper `Test.php` suffix